### ISSUE TITLE
Cheyenne-gnu build fails on some modules

### ISSUE THREAD

**Issue by BruceKropp-Raytheon** [NOAA-EMC/GSI#596]
While trying to build on Cheyenne with GNU compiler, it reports module failures.

Steps:
```
set -x
cd ush && MACHINE=cheyenne ./build.sh
```

Output:
```
+ cd ush
+ MACHINE=cheyenne
+ ./build.sh
++++ readlink -f -n ./build.sh
+++ dirname /glade/scratch/epicufsrt/jenkins/workspace/sandbox/emc-gsi-build/gsi/ush/build.sh
++ cd /glade/scratch/epicufsrt/jenkins/workspace/sandbox/emc-gsi-build/gsi/ush/..
++ pwd -P
+ readonly DIR_ROOT=/glade/scratch/epicufsrt/jenkins/workspace/sandbox/emc-gsi-build/gsi
+ DIR_ROOT=/glade/scratch/epicufsrt/jenkins/workspace/sandbox/emc-gsi-build/gsi
+ BUILD_TYPE=Release
+ CMAKE_OPTS=
+ COMPILER=gnu
+ BUILD_DIR=/glade/scratch/epicufsrt/jenkins/workspace/sandbox/emc-gsi-build/gsi/build
+ INSTALL_PREFIX=/glade/scratch/epicufsrt/jenkins/workspace/sandbox/emc-gsi-build/gsi/install
+ GSI_MODE=Regional
+ ENKF_MODE=GFS
+ REGRESSION_TESTS=YES
+ source /glade/scratch/epicufsrt/jenkins/workspace/sandbox/emc-gsi-build/gsi/ush/detect_machine.sh
++ case $(hostname -f) in
+++ hostname -f
++ MACHINE_ID=cheyenne
++ '[' cheyenne = hera ']'
++ '[' cheyenne = cheyenne ']'
++ MACHINE_ID=cheyenne.gnu
+ set +x
Lmod has detected the following error: These module(s) or extension(s) exist
but cannot be loaded as requested: "ncio/1.1.2", "w3emc/2.9.1", "bufr/11.7.0"
   Try: "module spider ncio/1.1.2 w3emc/2.9.1 bufr/11.7.0" to see how to load
the module(s).

Executing this command requires loading "bufr/11.7.0" which failed while
processing the following module(s):

    Module fullname   Module Filename
    ---------------   ---------------
    gsi_common        /glade/scratch/epicufsrt/jenkins/workspace/sandbox/emc-gsi-build/gsi/modulefiles/gsi_common.lua
    gsi_cheyenne.gnu  /glade/scratch/epicufsrt/jenkins/workspace/sandbox/emc-gsi-build/gsi/modulefiles/gsi_cheyenne.gnu.luaExecuting this command requires loading "w3emc/2.9.1" which failed while
processing the following module(s):

    Module fullname   Module Filename
    ---------------   ---------------
    gsi_common        /glade/scratch/epicufsrt/jenkins/workspace/sandbox/emc-gsi-build/gsi/modulefiles/gsi_common.lua
    gsi_cheyenne.gnu  /glade/scratch/epicufsrt/jenkins/workspace/sandbox/emc-gsi-build/gsi/modulefiles/gsi_cheyenne.gnu.luaExecuting this command requires loading "ncio/1.1.2" which failed while
processing the following module(s):

    Module fullname   Module Filename
    ---------------   ---------------
    gsi_common        /glade/scratch/epicufsrt/jenkins/workspace/sandbox/emc-gsi-build/gsi/modulefiles/gsi_common.lua
    gsi_cheyenne.gnu  /glade/scratch/epicufsrt/jenkins/workspace/sandbox/emc-gsi-build/gsi/modulefiles/gsi_cheyenne.gnu.lua
```

**Comment by DavidHuber-NOAA** [comment]
The Cheyenne GNU modulefile will be updated alongside the Intel 2022 upgrade (PR #571).

**Comment by RussTreadon-NOAA** [comment]
May this issue be closed?

**Comment by DavidHuber-NOAA** [comment]
Yes, I believe so.  This should have been resolved with #571 and besides that Cheyenne will retire at the end of this month.

**Comment by RussTreadon-NOAA** [comment]
OK.  I didn't realize Cheyenne is retiring at the end of 2023.   I'll close this and other Cheyenne related issues.
